.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,9 @@ Here's a handy checklist to ensure your PR goes smoothly.
 - [ ] My code conforms to Google's [python style](https://google.github.io/styleguide/pyguide.html).
 - [ ] My PR at a minimum doesn't decrease unit-test coverage (if applicable).
 - [ ] My PR has been functionally tested.
-- [ ] All of the [tests](https://forsetisecurity.org/docs/latest/develop/dev/testing.html) pass
+- [ ] All of the [tests](https://forsetisecurity.org/docs/latest/develop/dev/testing.html) pass.
+- [ ] I have submitted a corresponding PR for documentation in the `forsetisecurity.org-dev` branch (if applicable).
+- [ ] My feature has documentation that has been functionally used and tested (if applicable).
```

Comment:
How about a small tweak?

nit: `My documentation has been functionally tested (if applicable).`

Reply:
Should we ask them to provide a link in the PR here to the documentation PR?